src/main/java/bio/terra/janitor/service/cleanup/flight/GoogleBucketCleanupStep.java

```diff
@@ -0,0 +1,74 @@
+package bio.terra.janitor.service.cleanup.flight;
+
+import bio.terra.cloudres.common.ClientConfig;
+import bio.terra.cloudres.google.storage.StorageCow;
+import bio.terra.generated.model.CloudResourceUid;
+import bio.terra.janitor.db.JanitorDao;
+import bio.terra.janitor.db.TrackedResource;
+import bio.terra.janitor.db.TrackedResourceId;
+import bio.terra.janitor.db.TrackedResourceState;
+import bio.terra.janitor.service.cleanup.CleanupParams;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.StepStatus;
+import com.google.cloud.storage.StorageException;
+import com.google.cloud.storage.StorageOptions;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Step to cleanup Google Bucket resource. */
+public class GoogleBucketCleanupStep implements Step {
+  private Logger logger = LoggerFactory.getLogger(GoogleBucketCleanupStep.class);
+
+  private final StorageCow storageCow;
+  private final JanitorDao janitorDao;
+
+  public GoogleBucketCleanupStep(ClientConfig clientConfig, JanitorDao janitorDao) {
+    this.janitorDao = janitorDao;
+    // Janitor only uses CRL Cows to delete resources. Cleanup is not needed.
```

Comment:
comment's not appropriate here. this cleanupstep does not care how the ClientConfig is created.